### PR TITLE
Filtering out the repos which have all 0-s in the row

### PR DESCRIPTION
**Pull request by mkArtakMSFT**

### CODE REVIEW

**Comment by mkArtakMSFT** [comment]
@Eilon , can you please review this. Thanks !

File: src/Hubbup.Web/Views/IssueList/_RepoList.cshtml [hunk 1]
```diff
@@ -13,9 +13,10 @@
         </tr>
     </thead>
     <tbody>
+        @{var rowsRendered = 0; }
```
Comment (Eilon) [inline src/Hubbup.Web/Views/IssueList/_RepoList.cshtml]: Can wrap this `var` and the `if` in one big `@{ ... ... ... }`. Might be a little cleaner that way.

**Review by Eilon — CHANGES_REQUESTED** [review]

**Review by Eilon — APPROVED** [review]
Looks good!

**Comment by Eilon** [comment]
It will be a while until I deploy this live - I need to get my new laptop set up for deploying to Azure first.

**Comment by mkArtakMSFT** [comment]
No problem. Thanks !

**Pull request merged**